ParseUI/Classes/LogInViewController/PFLogInView.m

```diff
@@ -35,6 +35,16 @@
 static NSString *const PFLogInViewDefaultFacebookButtonImageName = @"facebook_icon.png";
 static NSString *const PFLogInViewDefaultTwitterButtonImageName = @"twitter_icon.png";
 
+#pragma mark - Accessibility labels identifiers
+NSString * const PFLogInViewUsernameFieldAccessibilityIdentifier = @"username";
```

Comment:
Nit:  Using ambiguous labels won't quite make it unique - I would recommend Cocoa approach to this aka use the same string as the name of the property.